package.json

```diff
@@ -9,7 +9,8 @@
       "packages/installer/*",
       "packages/maker/*",
       "packages/publisher/*",
-      "packages/utils/*"
+      "packages/utils/*",
+      "packages/plugin/*"
```

Comment:
plugin or plugins?

Reply:
`plugin`, it's consistent with `installer` and `maker`

Reply:
`util` perhaps :smile:

Reply:
Yup will follow up on that